uelc/settings_shared.py

```diff
@@ -196,7 +196,7 @@
 
 SESSION_ENGINE = "django.contrib.sessions.backends.signed_cookies"
 SESSION_COOKIE_HTTPONLY = True
-SESSION_SERIALIZER = 'django.contrib.sessions.serializers.JSONSerializer'
+SESSION_SERIALIZER = 'django.contrib.sessions.serializers.PickleSerializer'
```

Comment:
`PickleSerializer` is a potential security issue (possibility of remote code execution if the `SECRET_KEY` leaks).